app/api/_services/user.ts

```diff
@@ -76,10 +82,17 @@ export const createUser = async (user: CreateUserProp) => {
       profile: newProfile,
       rolesId: [newRole._id],
     });
-    RoleModel.findByIdAndUpdate(newRole._id, {
+
+    await RoleModel.findByIdAndUpdate(newRole._id, {
       ownerId: newUser._id,
     }).exec();
-    // TODO: Return the JWT token
+
+    await sendbirdRequests.createUser({
+      user_id: newRole._id,
+      nickname: newProfile.displayName,
+      profile_url: "https://sendbird.com/main/img/profiles/profile_05_512px.png",
+      issue_access_token: true,
+    });
```

Comment:
It seems like you created a sendbird user during the Cearte User. But the sendbird user should be linked to the Role. As the role is the source for the chat.

Reply:
The user creation function automatically assigns a default role. Therefore, I think it is advisable to create a new Sendbird user using the ID of the newly assigned role.